cvprac/cvp_client.py

```diff
@@ -170,7 +171,7 @@ def set_log_level(self, log_level='INFO'):
         self.log.setLevel(getattr(logging, log_level))
 
     def connect(self, nodes, username, password, connect_timeout=10,
-                protocol='http', port=None):
+                port=None, cert=False):
```

Comment:
I have a love hate relationship with python. Dynamic typing is nice, but in this case it's confusing. When I see cert=False, I think cert is a bool...but cert can also be a path to cert file. So maybe update the comment to reflect that this could be either a boolean or a string.